clients/geth/specular/rollup/services/sequencer/sequencer.go

```diff
@@ -277,8 +276,9 @@ func (s *Sequencer) sequencingLoop(genesisRoot common.Hash) {
 				commitAssertion()
 			}
 			batchTxs = nil
-		case batch := <-s.batchCh:
+		case blocks := <-s.blockCh:
 			// New batch from Batcher
+			batch := rollupTypes.NewTxBatch(*blocks, 0) // TODO: add max batch size
```

Comment:
shouldn't create batches here; instead please push these blocks into a local array. create batches in ticks